data/index.html

```diff
@@ -136,38 +136,48 @@ <h5 class="text-center">USGS Topo Maps</h5>
+        <p class="readable">The Open SGID is a cloud-hosted PostGIS database containing our curated collection of GIS datasets. It is a publicly-available database connection that can be used in many GIS platforms to directly access Utah spatial data. You can find the most up-to-date information about the Open SGID, including troubleshooting tips and an issue tracker, in the <a href="https://github.com/agrc/open-sgid">GitHub repository</a>. Please read, understand, and abide by the <a href="{% link about/policy/open-sgid/index.md %}">terms of use</a> prior to connecting.</p>
         <p class="section-read-more">
-            <a href="{% link sgid/open-sgid/index.md %}" class="btn">Get connected</a>
+            <a href="{% link sgid/open-sgid/index.md %}" class="btn">Get Connected</a>
         </p>
     </div>
-    <div class="grid__col section alternate">
-    <h2>SGID Index</h2>
-        <div class="grid__col--span-all grid__col--centered grid__col--s-hidden" style="width:568px">
-            <a href="{% link data/sgid-index/index.html %}">
-                <img alt="SGID Index Table Search" class="text-center" src="{% link images/sgid-index.png %}">
-            </a>
-        </div>
-        <p class="readable">The SGID Index is a sortable and filterable table containing data items contained in our SGID. It is a great resource to quickly find GIS data and resources important to you. In the SGID Index, you will find the ISO category, the name of the resource, the agency who is the data steward, a short description and a link to a service endpoint if one is available. </p>
+    <div class="grid__col section">
+        <h2>The SGID Open Data Site</h2>
+        <p class="readable">The <a href="https://opendata.gis.utah.gov">SGID Open Data site</a> leverages Esri's Open Data platform to create a single portal for any GIS resources hosted on Esri's ArcGIS Online or Enterprise Portal solutions. Each item in Open Data contains web service links for consuming the datasets in webmaps and other applications along with direct data download links. We have uploaded all our curated datasets to ArcGIS Online and shared them with the site, and we encourage other agencies and organizations to <a href="{% link gis-data-sharing/open-data/index.md %}">share</a> their datasets with us as well.</p>
```

Comment:
**thought**: i'm not sure i like the `with us` piece

> we encourage other agencies and organizations to <a href="{% link gis-data-sharing/open-data/index.md %}">share</a> their datasets with us as well.

What about something like

> we encourage other agencies and organizations to share their datasets through the Open SGID site.

> we encourage other agencies and organizations to share their datasets with us through the Open SGID site.

We might add some encouraging verbiage like 

> we encourage other agencies and organizations to expand the usefulness of the SGID and contribute to open data by sharing their data.

I'm not sure i like those words but you know what I mean hopefully

Reply:
Yeah, I see what you're getting at. 

I'm also realizing that I forgot the mention ArcGIS Online (except tangentially in the Open Data section) and because our data will show up in searches in ArcGIS Pro/Online, I think we need to mention that, which also clears up the other comment about uploading and sharing. New commit forthcoming.